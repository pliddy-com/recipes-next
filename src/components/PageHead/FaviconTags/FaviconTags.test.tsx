import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import FaviconTags from './FaviconTags';

describe('FaviconTags', () => {
  // reset mocks after each test
  afterEach(() => {
    jest.resetModules();
  });

  describe('when there is a config object', () => {
    it('it renders a Next Head tag with a title and description', () => {
      const { asFragment } = render(<FaviconTags />);

      expect(asFragment()).toMatchSnapshot();
    });
  });
});
