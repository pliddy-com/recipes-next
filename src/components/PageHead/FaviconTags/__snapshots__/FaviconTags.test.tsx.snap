// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FaviconTags when there is a config object it renders a Next Head tag with a title and description 1`] = `
<DocumentFragment>
  <link
    href="/apple-touch-icon.png?v=2"
    rel="apple-touch-icon"
    sizes="180x180"
  />
  <link
    href="/favicon-32x32.png?v=2"
    rel="icon"
    sizes="32x32"
    type="image/png"
  />
  <link
    href="/favicon-16x16.png?v=2"
    rel="icon"
    sizes="16x16"
    type="image/png"
  />
  <link
    href="/site.webmanifest?v=2"
    rel="manifest"
  />
  <link
    color="hsl(205, 60%, 25%)"
    href="/safari-pinned-tab.svg?v=2"
    rel="mask-icon"
  />
  <link
    href="/favicon.ico?v=2"
    rel="shortcut icon"
  />
  <meta
    content="Recipes"
    name="apple-mobile-web-app-title"
  />
  <meta
    content="Recipes"
    name="application-name"
  />
  <meta
    content="hsl(205, 60%, 25%)"
    name="msapplication-TileColor"
  />
  <meta
    content="#fff"
    name="theme-color"
  />
</DocumentFragment>
`;
